financialaid/management/commands/generate_exchange_rates.py

```diff
@@ -0,0 +1,16 @@
+"""
+Generates CurrencyExchangeRate objects
+"""
+from django.core.management import BaseCommand
+
+from financialaid.tasks import sync_currency_exchange_rates
+
+
+class Command(BaseCommand):
+    """
+    Generates CurrencyExchangeRate objects by calling sync_currency_exchange_rates
+    """
+    help = "Generates CurrencyExchangeRate objects by calling sync_currency_exchange_rates"
```

Comment:
I'd also suggest naming this file `update_exchange_rates.py`, so that the command to use it is `./manage.py update_exchange_rates`.